session/pessimistic_test.go

```diff
@@ -60,9 +60,11 @@ func (s *testPessimisticSuite) SetUpSuite(c *C) {
 	s.dom, err = session.BootstrapSession(s.store)
 	s.dom.GetGlobalVarsCache().Disable()
 	c.Assert(err, IsNil)
+	tikv.PrewriteMaxBackoff = 500
```

Comment:
Does this lead to data race?

Reply:
Yes, I was hoping for luck  😂 
And I put this change in SetUpSuite to reduce the data race probability.